lib/lmi_client.rb

```diff
@@ -0,0 +1,31 @@
+class LmiClient
+  BASE_URL = 'http://api.lmiforall.org.uk/api/v1/'
```

Comment:
HTTParty has built-in support for base_uris etc https://github.com/jnunemaker/httparty#examples